.github/workflows/stacks-blockchain-tests.yml

```diff
@@ -53,48 +53,23 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     steps:
-      - name: Add code coverage tools
-        run: |
-          rustup component add llvm-tools-preview
-          cargo install grcov
-      - name: Add nextest
-        run: |
-          cargo install cargo-nextest
+      - name: Add nextest and cargo-llvm-cov
+        uses: taiki-e/install-action@v2
```

Comment:
What's this?

Reply:
An action to easily and quickly install rust tools. I tested it on clarinet and it's quite fast